### DISCUSSION TITLE
Nextcloud AIO Install - network name nextcloud-aio already used: network already exists error

### DISCUSSION THREAD

**Discussion by kri164** [nextcloud/all-in-one#1304] [Questions] [answered]
### Steps to reproduce
1. Start AIO container as described on https://github.com/nextcloud/all-in-one#how-to-use-this
2. Log in to localhost:8080

### Expected behavior 
It should working.

### Actual behavior 
Got the same problem, as reported by user ferreo in
https://help.nextcloud.com/t/nextcloud-aio-install-network-name-nextcloud-aio-already-used-network-already-exists-error/144972

### Host OS 
openSUSE Tumbleweed 20221022 and openSUSE Leap 15.3

#### Nextcloud AIO version 
REPOSITORY                  TAG       IMAGE ID       CREATED       SIZE
nextcloud/all-in-one        latest    fb706096933c

#### Other valuable info 
Will be filled on request.

Thank you

**Comment by szaimen** [comment]
Hi, can you post the exact command that you used to create the mastercontainer?

**Comment by blackmore190** [comment]
Is this a second attempt to set everything up? Have you followed all the steps from [here](https://github.com/nextcloud/all-in-one#how-to-properly-reset-the-instance) before your second try?

especially this point: "Delete the docker network with sudo docker network rm nextcloud-aio"

**Comment by kri164** [comment, marked as answer]
<details>
<summary>details</summary>


```
# whoami
root
```

```
# docker network list
NETWORK ID     NAME      DRIVER    SCOPE
2f259bab93aa   bridge    bridge    local
```

```
# docker run \
--sig-proxy=false \
--name nextcloud-aio-mastercontainer \
--restart always \
--publish 80:80 \
--publish 8080:8080 \
--publish 8443:8443 \
--volume nextcloud_aio_mastercontainer:/mnt/docker-aio-config \
--volume /var/run/docker.sock:/var/run/docker.sock:ro \
nextcloud/all-in-one:latest

```

</details>

Seems the docker daemon process was dead or unhealthly.
This problem was fixed by running command:

**systemctl restart docker**

Thank you very much for your great and quick respond.

**Comment by szaimen** [comment]
I'll move this to discussions so that other people find your solution.
